device-pkgs.nix

```diff
@@ -170,16 +170,16 @@ let
   # See l4t_generate_soc_bup.sh
   # python ${edk2-jetson}/BaseTools/BinWrappers/PosixLike/GenerateCapsule -v --encode --monotonic-count 1
   # NOTE: providing null public certs here will use the test certs in the EDK2 repo
-  mkUefiCapsuleUpdate = lib.makeOverridable ({
-    trustedPublicCertPemFile ? null,
-    otherPublicCertPemFile ? null,
-    signerPrivateCertPemFile ? null,
-    requiredSystemFeatures ? [ ],
-  }: runCommand "uefi-${hostName}-${l4tVersion}.Cap" { nativeBuildInputs = [ python3 openssl ]; inherit requiredSystemFeatures; } ''
+  uefiCapsuleUpdate = runCommand "uefi-${hostName}-${l4tVersion}.Cap" {
+    nativeBuildInputs = [ python3 openssl ];
+    inherit (cfg.firmware.secureBoot) requiredSystemFeatures;
```

Comment:
I believe this `requiredSystemFeatures` came from `cfg.firmware.uefi.capsuleAuthentication.requiredSystemFeatures`

Reply:
Fixed in new commit